app/app.go

```diff
@@ -866,25 +866,25 @@ func (app *App) InstanceEnv(name string) map[string]bind.EnvVar {
 
 // Run executes the command in app units, sourcing apprc before running the
 // command.
-func (app *App) Run(cmd string, w io.Writer, once bool) error {
+func (app *App) Run(cmd string, w io.Writer, once, isolated bool) error {
```

Comment:
One small nitpick, it might be better to create a new type, something like RunArgs, to group this parameters and send them along the way to `sourced` and `run`. Having two booleans in the signature may start getting messy in the future and lead to bugs where the two are swapped by mistake.